args:
  dtype:
  - tensor
  - int
  - tensor
  is_pos:
  - false
  - false
  - false
  name:
  - a
  - decimals
  - out
  required:
  - true
  - false
  - false
name: numpy.round
package: null
pass_rate: 13.666666666666666
rules:
- - cot: 'The error is due to an invalid output shape. It seems that the shape of
      the output tensor ''out'' is not compatible with the function numpy.round. In
      numpy.round, the output shape is determined by the input tensor ''a''. Therefore,
      the shapes of tensor ''a'' and ''out'' should be the same. The condition that
      makes the error disappear is:'
    length: 1
    target:
      choosen_dtype:
        a: tensor
        decimals: int
        out: tensor
      msg: invalid output shape
      package: numpy
    txt: a.shape == out.shape
  - f1_score: 68.70897155361051
    overall_score: 29.166666666666664
    precision: 52.33333333333333
    recall: 100.0
